challenge.md

```diff
@@ -21,6 +34,9 @@
   </section>
 </main>
 ```
+    **- const h2 = document.querySelector(.subtitle) 
+    h2.textContent = "The MERN stack"
+    h2.ClassList.remove("subtitle")**
```

Comment:
This script will not work as expected. Have you tested it?

`ClassList` is not a property of HTMLElement

**FIX**: define correctly class list method